Packs/Code42/Integrations/Code42/README.md

```diff
@@ -18,11 +18,11 @@ Use the Code42 integration to identify potential data exfiltration from insider
 
 4. Click **Test** to validate the URLs, token, and connection.
 ## Commands
-You can execute these commands from the Demisto CLI, as part of an automation, or in a playbook.
```

Comment:
Even though this is correct, this part particularly is not really controlled by us